### PR TITLE
Adding Brave Support

### LINKED ISSUE #12: Support for other browsers
First of all, thanks for creating this workflow, it has been really helpful!

I wanted to know what are your thoughts on extending the config file to support more Chromium-based browsers? 

I use Brave Browser, so I had to fork this repo and tweak the code a bit to make it work with Brave. 
Instead of just adding a new URL to `getDBFilePath` I figured I could make it even more flexible if I just included a config property that allowed me to specify the browser support directory. 

You can check my commit [here](https://github.com/hawkeng/alfred-chrome-workflow/commit/2503083a953d5a64ab8e43e789576cd56afc3793).

If you agree and are okay with the changes, I can create a PR, otherwise thanks for your time.

### PR DESCRIPTION
**Pull request by jun6lee**
Fixes #12.

Through Config change, can support Brave. 
(Tested on MacOS 12.3.1)

### CODE REVIEW

**Review by jopemachine — CHANGES_REQUESTED** [review]
Thanks for the PR!
I think it is great to have `browser_dir` option for other browser support like [this one](https://github.com/hawkeng/alfred-chrome-workflow/commit/2503083a953d5a64ab8e43e789576cd56afc3793).

Could you add this logic?
I think because the logic is `optional`, so you don't need to update `src/init.js` and other codes.
But README needs to mention the `browser_dir` option.

**Comment by jun6lee** [comment]
Done. 👍️

File: src/utils.js [hunk 1]
```diff
@@ -89,16 +89,11 @@ const getDBFilePathWithConf = (DBFile) => {
...
-  switch (conf['browser']) {
-  case 'Chrome Canary':
-    return `/Users/${userName}/Library/Application Support/Google/Chrome Canary/${chromeProfilePath}/${DBFile}`;
-  case 'Edge':
-    return `/Users/${userName}/Library/Application Support/Microsoft Edge/${chromeProfilePath}/${DBFile}`;
-  case 'Chromium':
-    // 'Chrome Cloud Enrollment' could be wrong (not sure)
-    return `/Users/${userName}/Library/Application Support/Google/Chrome Cloud Enrollment/${chromeProfilePath}/${DBFile}`;
-  default:
-    return `/Users/${userName}/Library/Application Support/Google/Chrome/${chromeProfilePath}/${DBFile}`;
```
Comment (jopemachine) [inline src/utils.js:101]: We cannot remove the codes here.
When `browser_dir` is not specified, `getDBFilePath` return undefined. (`browser_dir` is optional)

File: src/utils.js [hunk 2]
```diff
@@ -89,16 +89,11 @@ const getDBFilePathWithConf = (DBFile) => {
...
+
+  let browserDir = conf['browser_dir'];
+  if (typeof browserDir === 'string' && browserDir.trim()) {
+    browserDir = browserDir.trim().replace(/\/$/, '');
+    return `/Users/${userName}/Library/Application Support/${browserDir}/${chromeProfilePath}/${DBFile}`;
```
Comment (jopemachine) [inline src/utils.js]:
```suggestion
  let browserDir = conf['browser_dir'];
  if (typeof browserDir === 'string' && browserDir.trim()) {
    browserDir = browserDir.trim().replace(/\/$/, '');
    return `${browserDir}/${chromeProfilePath}/${DBFile}`;
  }

  switch (conf['browser']) {
  case 'Chrome Canary':
    return `/Users/${userName}/Library/Application Support/Google/Chrome Canary/${chromeProfilePath}/${DBFile}`;
  case 'Edge':
    return `/Users/${userName}/Library/Application Support/Microsoft Edge/${chromeProfilePath}/${DBFile}`;
  case 'Chromium':
    // 'Chrome Cloud Enrollment' could be wrong (not sure)
    return `/Users/${userName}/Library/Application Support/Google/Chrome Cloud Enrollment/${chromeProfilePath}/${DBFile}`;
  default:
    return `/Users/${userName}/Library/Application Support/Google/Chrome/${chromeProfilePath}/${DBFile}`;
  }
```

File: src/utils.js (hunk 1 above)
Comment (jun6lee) [reply src/utils.js:101]: Or you could set "browser_dir" in config with a default value of "Chrome Path", which is what I went with as it seemed *more efficient*. We can always set it as a default and still have changing it as optional, much like all the other config items.

Comment (jopemachine) [reply src/utils.js:101]: Then users should know the DB filepath, not just the browser name. (DB filepath here varies slightly from browser to browser.)
This is breaking change and I don't see it as more efficient.
And it doesn't make sense `getDBFilePath` might return `undefined`.

File: src/utils.js (hunk 2 above)
Comment (jun6lee) [reply src/utils.js]: OK

**Comment by jopemachine** [comment]
Thanks for your PR :)

**Pull request merged**